rim/views.py

```diff
@@ -21,6 +27,36 @@ def get_ordering(self):
         else:
             return [Lower(self.order).asc()]
 
+    def get(self, request, *args, **kwargs):
+        if self.export_csv:
+            keys = ['serial_no', 'equipment_model', 'manufacturer', 'equipment_type__type_name', 'latest_checkout__location__building', 'latest_checkout__location__room']
+            verbose_keys = []
+            for key in keys:
+                split_key = key.split('__')
+                foreign_key = Equipment._meta.get_field(split_key[0])
+                while True:
+                    if len(split_key) > 1:
+                        foreign_key = foreign_key.remote_field.model._meta.get_field(split_key[1])
+                        split_key = split_key[1:]
+                    else:
+                        verbose_keys.append(foreign_key.verbose_name)
+                        break
+
+            equipment_list = self.get_queryset().values(*keys)
+            current_time = localtime(now()).strftime("%a-%b-%d-%I%M")
+            print(current_time)
+            response = HttpResponse(content_type='text/csv')
+            response['Content-Disposition'] = 'attachment; filename="' + current_time + '.csv"'
```

Comment:
We should probably still prefix the timestamp with some kind of title. "RIM report" or something.